### PR TITLE
Add Footer

### PR DESCRIPTION
**Pull request by manuelpapa**
Added a responsive Footer to Storybook.
image

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -11,7 +14,12 @@ function App() {
   return (
     <>
       <GlobalStyles />
-      <p>Hola Papa!</p>
+      <h2>Headline</h2>
```
Comment (mpagels) [inline client/src/App.js:17]: You cannot have a h2 tag unless there's a h1 tag beforehand. is it to big? style id ;-)

```suggestion
      <h1>Headline</h1>
```

File: client/src/components/InputField.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  border: none;
+  border-bottom: 1px solid var(--font-semi-dark);
+  ::placeholder {
+    color: var(--font-semi-dark);
+    font-size: 13px;
+    text-align: center;
+  }
+`;
+
+// export default function SearchInput({ value, onChange }) {
```
Comment (mpagels) [inline client/src/components/InputField.js:17]:
```suggestion
```

File: client/src/components/InputField.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export default function SearchInput() {
+  return (
+    <Input
+      placeholder="Eingabe"
+      // value={value}
+      // onChange={(event) => onChange(event.target.value.trim())}
```
Comment (mpagels) [inline client/src/components/InputField.js:23]:
```suggestion
```

File: client/src/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  }
+`;
+
+function Footer() {
+  return (
+    <FooterContainer>
+      <FooterGradient />
+      <FooterIcons>
+        <Link to="/AddIcon">
+          <img src={AddIcon} alt="Essen teilen" />
```
Comment (mpagels) [inline client/src/components/Footer.js:43]: AddIcon and "Essen teilen" seems confusing. Do you add a meal or do you share a meal?

File: client/src/components/Footer.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        </Link>
+        <Link to="/ListIcon">
+          <img src={ListIcon} alt="Liste durchsuchen" />
+        </Link>
+        <Link to="/UserIcon">
+          <img src={UserIcon} alt="Mein Bereich" />
```
Comment (mpagels) [inline client/src/components/Footer.js:49]: alt value could be better. what "mein bereich" means? your profile area? your saved meals area?

File: client/src/App.js [hunk 6]
```diff
@@ -11,7 +14,12 @@ function App() {
...
+      <h3>Title tile</h3>
```
Comment (mpagels) [inline client/src/App.js:18]:
```suggestion
      <h2>Title title</h2>
```

File: client/src/App.js [hunk 7]
```diff
@@ -11,7 +14,12 @@ function App() {
...
+      <p>Text Text</p>
+      <InputField>E-Mail</InputField>
+      <SubmitButton>Login</SubmitButton>
+      <small>Register</small>
     </>
```
Comment (mpagels) [inline client/src/App.js:23]: Pull request named Add Footer. Footer is missing in App.js?

File: client/src/components/SubmitButton.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  border: 0;
+  border-radius: 3em;
+  padding: 1em 2em;
+  display: inline-block;
+  line-height: 1;
+  background-image: var(--bg-main-gradient);
+  cursor: pointer;
+`;
+
+export default SubmitButton;
```
Comment (mpagels) [inline client/src/components/SubmitButton.js:16]: Would rename the file to Button. So you can reuse this button better.

**Review by mpagels — COMMENTED** [review]
storybook stories seems ok. good job on this one. was a little bit confused with this pull request. but i think i get it now. you working "ticket" like and not story like.

found some other tiny things.

File: client/src/components/SubmitButton.js (hunk 8 above)
Comment (manuelpapa) [reply client/src/components/SubmitButton.js:16]: Thank you, you're right!

File: client/src/App.js (hunk 7 above)
Comment (manuelpapa) [reply client/src/App.js:23]: As you already mentioned, it's accessible in the storybook but not yet in the App.js.

File: client/src/components/Footer.js (hunk 5 above)
Comment (manuelpapa) [reply client/src/components/Footer.js:49]: True. Will think about a better naming though.

**Pull request merged**